data/hails.txt

```diff
@@ -1824,60 +1841,14 @@ phrase "friendly pirate"
 		" I wish I could afford more"
 		" I'm going to need to make money to buy more"
 		" Best experience I've ever had"
+		" It's unreal."
+		" It'll make you forget you're alive."
+		" It'll make you soar."
 	word
 		"!"
 		"."
 
-phrase "friendly pirate"
-	word
-		"Have you "
-	word
-		"tried this"
-		"used this"
-	word
-		" new "
-		" terrible "
-		" horrendous "
-		" atrocious "
-		" horrid "
-		""
-	word
-		"bad blue"
-		"booster"
-		"core"
-		"death stick"
-		"deep"
-		"fire apple"
-		"flavor drop"
-		"good red"
-		"hot wing"
-		"memory chip"
-		"quasar"
-		"red giant"
-		"ringworld shaving"
-		"solaris joyride"
-		"southern snack"
-		"space weed"
-		"stardust"
-		"steam"
-		"toad juice"
-		"whale blubber"
-	word
-		" "
-	word
-		"stuff?"
-		"drug?"
-	word
-		""
-		" I couldn't see for a week"
-		" I couldn't get out of bed for a week"
-		" It felt like I was being eaten alive"
-		" It felt like I had insects crawling under my skin"
-		" I'm never taking that stuff again"
-		" Worst experience I've ever had"
-	word
-		"!"
-		"."
+#duplicate entry removed, see above.	
```

Comment:
This wasn't a duplicate. The above hails mention how the drugs are good while the hails you deleted were the ones giving the drugs bad reviews.
`"Have you used this incredible stardust drug? It's the best!"`
vs a possible combo of what you deleted
`"Have you used this horrendous stardust drug? It felt like I was being eaten alive!"`